testing/ncids-js-testing/cypress/integration/InnerPage/inner-page.js

```diff
@@ -0,0 +1,6 @@
+import { Given } from 'cypress-cucumber-preprocessor/steps';
+
+const url = 'http://localhost:8080/inner-pages/inner-page.html';
```

Comment:
cy.visit() automatically appends path to a baseUrl set in config, so you do not need to store full url in the js files, just use a path param you pass through the step
step below becomes 
Given('user navigates to {string}',(path)=>{
cy.visit(path);
}